### PR TITLE
Will development

### PR DESCRIPTION
**Pull request by dunlapww**
Add setup for GameTeam and GameTeam manager
All tests passing

### CODE REVIEW

File: lib/games_manager.rb [hunk 1]
```diff
@@ -46,8 +46,9 @@ def percentage_visitor_wins
     ratio(wins, total_games)
   end
 
-  def total_games
-    @games.count
+  def total_games(filtered_games = @games)
+    filtered_games.count
   end
```
Comment (jdcarey128) [inline lib/games_manager.rb:51]: Let's keep total_games stripped down for now. Use the method season_group to count games by season.

**Review by jdcarey128 — CHANGES_REQUESTED** [review]

**Review by jdcarey128 — APPROVED** [review]

File: lib/games_manager.rb [hunk 2]
```diff
@@ -46,8 +46,46 @@ def percentage_visitor_wins
...
+  end
+
+
+
+
+
+
+
+
+
```
Comment (hopesgit) [inline lib/games_manager.rb]: Lots of empty space here.

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@ def initialize(locations)
     load_managers(locations)
   end
 
-  def self.from_csv(locations = {games: './data/games_sample.csv', teams: './data/teams_sample.csv', game_teams: './data/game_teams_sample.csv'})
+  def self.from_csv(locations = {games: './data/games.csv', teams: './data/teams_sample.csv', game_teams: './data/game_teams_sample.csv'})
```
Comment (hopesgit) [inline lib/stat_tracker.rb]: Why run the full games array? That's gonna be *really slow* to test with

File: test/games_manager_test.rb [hunk 4]
```diff
@@ -28,5 +28,34 @@ def test_it_can_get_percentage_visitor_games_won
...
+    assert_equal 3.2, @games_manager.average_visitor_score
+  end
+
+  def test_it_can_get_visitor_w_min_avg_score
+    require "pry"; binding.pry
+    assert_equal 24, @games_manager.min_visitor_score
+  end
+
+
+
```
Comment (hopesgit) [inline test/games_manager_test.rb]: Empty space again

**Comment by hopesgit** [comment]
Solid work, Will!

**Review by hopesgit — CHANGES_REQUESTED** [review]
Just marking this for changes

File: test/games_manager_test.rb (hunk 4 above)
Comment (dunlapww) [reply test/games_manager_test.rb]: Thanks Hope, removed the empty space.

File: lib/games_manager.rb (hunk 1 above)
Comment (dunlapww) [reply lib/games_manager.rb:51]: Joshua, I need these arguments to calculate my averages on a varying set of game records.  If you have an alternate solution, i'm open to ideas.

File: lib/stat_tracker.rb (hunk 3 above)
Comment (dunlapww) [reply lib/stat_tracker.rb]: Good catch, sorry.  Was validating that i got correct spec answer.  Moved back to games_sample

File: lib/games_manager.rb (hunk 2 above)
Comment (dunlapww) [reply lib/games_manager.rb]: removed empty space. thanks

**Comment by hopesgit** [comment]
Looks good in terms of the things you fixed, but it had merge conflicts. You'll have to fix those.

**Pull request closed without merging**